src/main/java/com/stripe/model/Plan.java

```diff
@@ -42,7 +39,7 @@
 
   /** Same as `amount`, but contains a decimal value with at most 12 decimal places. */
   @SerializedName("amount_decimal")
-  BigDecimal amountDecimal;
+  String amountDecimal;
```

Comment:
What is this diff?

Reply:
Something we need to fix in the prettier generator. Some fields are defined as `type: string` in the OpenAPI spec, but with an additional `format: decimal` annotation. Such fields should be typed as `BigDecimal` in stripe-java instead of `String`.